docs/pvcscale.md

```diff
@@ -0,0 +1,179 @@
+# PVCScale Workload
+
+The pvcscale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload is designed to stress storage backend from the scale point of
+view. It will create desired number of pods and attach an PVC to each pod.
+During this test we measure time necessary for pods to be in `Ready/Ready` state
```

Comment:
What is `Ready/Ready` state?

Reply:
READY (1/1) - passed successfully  all liveness and readiness probes and started , will term READY instead of Ready/Ready